### PR TITLE
Image quality assessment

### PR DESCRIPTION
**Pull request by N950**
This is an example of image quality assessment experiment running directly from OAK RGB camera

The model outputs 4 quality classes
* Clean
* Blur
* Occlusion
* Bright

### CODE REVIEW

**Comment by themarpe** [comment]
@N950 
Any reason for using an older depthai library? I suggest bumping it up to 2.15

**Comment by N950** [comment]
@themarpe 
Thank you, just changed it and tested with 2.15

File: image-quality-assessment/main.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+'''
+
+# --------------- Arguments ---------------
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-nn', '--nn_path', type=str, help="select model blob path for inference, defaults to image_quality_assessment_256x256_001 from model zoo", default=None)
+
+args = parser.parse_args()
+
+NN_PATH = args.nn_path if args.nn_path else str(blobconverter.from_zoo(name="image_quality_assessment_256x256", zoo_type="depthai"))
```
Comment (tersekmatija) [inline image-quality-assessment/main.py:25]: I am working with @VanDavv to get this working.

**Review by tersekmatija — CHANGES_REQUESTED** [review]
Could you also rename `image-quality-assessment` to `gen2-image-quality-assessment`, so it's aligned with other models

**Review by tersekmatija — APPROVED** [review]
Thanks!

**Comment by tersekmatija** [comment]
Thanks for that last fix!

**Pull request merged**